List/src/main/java/com/list/TxtFileLog.java

```diff
@@ -0,0 +1,136 @@
+package com.list;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+* <h1>TxtFileLog!</h1>
+*
+* The TxtFileLog class implements an application that performs operations on files.
+*
+* @author  Vitor Emanuel
+* @version 1.0
+* @since   2022-12-01
+*/
+public class TxtFileLog {
+    
+  private FileReader arq;
+  private BufferedReader inFile;
+  private String nameOutFile;
+  private BufferedWriter outFile;
+
+  /**
+  * This is the constructor with parameters using the standard
+  * encryption/decryption key, in this all variables are initialized as passed.
+  * 
+  * @param fileIn This is a String representing the input file.
+  * @param fileOut This is a String representing the output file.
+  */
+  public TxtFileLog(String fileIn, String fileOut){
+      openInFile(fileIn);
+      startOutFile(fileOut);
+  }
+  public TxtFileLog(){
+      openInFile("./text_files/original.txt");
+      startOutFile("./text_files/encrypted.txt");
+  }
+  /**
+  * This method is used to initialize the output file, preparing it for future writes.
```

Comment:
```suggestion
  * startOutFile is used to initialize the output file, preparing it for future writes.
```
I'd suggest removing `This method` from all these doc comments, since it does not add any information.